evalml/tests/component_tests/test_text_featurizer.py

```diff
@@ -38,12 +35,9 @@ def test_featurizer_only_text(text_df):
                               'MEAN_CHARACTERS_PER_WORD(col_2)',
                               'POLARITY_SCORE(col_1)',
                               'POLARITY_SCORE(col_2)'])
-    for i in range(15):
-        expected_col_names.add(f'PART_OF_SPEECH_COUNT(col_1)[{i}]')
-        expected_col_names.add(f'PART_OF_SPEECH_COUNT(col_2)[{i}]')
     X_t = tf.transform(X)
     assert set(X_t.columns) == expected_col_names
-    assert len(X_t.columns) == 40
+    assert len(X_t.columns) == 10
```

Comment:
Wow, so part of speech was creating 30 features on this dataset?

Reply:
Yep, one feature for each part of speech